redis.conf

```diff
@@ -929,7 +930,7 @@ lua-time-limit 5000
 # cluster-require-full-coverage yes
 
 # This option, when set to yes, prevents replicas from trying to failover its
-# master during master failures. However the master can still perform a
+# master during master failures. However the replica can still perform a
```

Comment:
are you certain this is right?

Reply:
Yes. I did a test manually just now with the following steps.

step1: Create a cluster with all 16384 slots allocated to different shards. Each shard has a 1 master and 1 replica.
step2: Manually set cluster-require-full-coverage to `yes` of all the nodes.
step3: Choose a shard, and delete a allocated slot from both the master and replica.
step4: Use `redis-cli --cluster check <ip:port>` to verify that not all 16384 slots are covered by nodes.
step5: Do `CLUSTER FAILOVER` upon replica, and it succeeded with a response `OK`. And the checking of using `ROLE` also showed that it really succeeded.
step6: Use `redis-cli --cluster check <ip:port>` to check and it showed that not all 16384 slots are covered by nodes.

Reply:
We are allowed to do CLUSTER FAILOVER upon replicas only.
And cluster-replica-no-failover can only prevent non-manual failover but not manual failover.
FYI https://github.com/redis/redis/blob/9ef8d2f67155d6aa69b8344fc0b63e6dab8ce9f8/src/cluster.c#L3113